### DISCUSSION TITLE
Cannot Upload a Featured Image

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#150] [Q&A]
Hi Taufik, I cannot upload a featured image to one page. It's persist old error from the panel and I cannot save the image. Can you test and check this itself?

Thanks in Advance!

**Comment by taufik-nurrohman** [comment]
Look like this is caused by one thing.

My advice is to enable debug mode: Open `./index.php` and search for `define('DEBUG', false)` code. Replace the value to `true`, then start the test again.

If you see some error messages from the PHP script, please post it here!

**Comment by ghost** [comment]
Hi Taufik, now I can add an image. In the Frontend of my website is the image not visible. Under https://mecha-cms.de/article/start-homepage-mecha-cms is my new article with a featured image(the image is not showing). Can you check this? Thanks.

2: Can you please in your store website replace my link from internetblogger.de to https://liebrechts-portfolio.de . Internetblogger.de is not longer my actually blog and the Admin is another person. Thanks.

**Comment by taufik-nurrohman** [comment]
How did you add the featured image? Via [`panel.image`](https://github.com/mecha-cms/x.panel.image) or via image uploader in [`c-k-editor.4`](https://github.com/mecha-cms/x.c-k-editor.4)?

By the way, the [`layout.normal`](https://github.com/mecha-cms/layout.normal) don&rsquo;t have support for custom featured image. You will need to add the featured image block yourself.

[`layout.outdoor`](https://github.com/mecha-cms/layout.outdoor) do have this feature by default.
